ui/src/configs/shortcutsKeyboard.ts

```diff
@@ -0,0 +1,6 @@
+const dictionary = {
+    "QuerySubmit" : [{metaKey : true, code:"Enter"}, {ctrlKey:true, code:"Enter"}]
```

Comment:
what are metaKey and ctrlKey?

Reply:
the properties on the KeyboardEvent  and "rules" which I compare against